README.md

```diff
@@ -46,7 +46,7 @@ You can obtain our latest single JAR distribution from the GitHub releases and s
 ### Requirements
 
 In order to build the project from source the following dependencies must be installed on your system:
-- java (>= 8, 11 is recommended)
+- java (11, 17 are recommended)
```

Comment:
should we recommend to use https://adoptopenjdk.net/ ? rather than the java.com where it's a pain to find the version > 8

Reply:
Here the readme is more for devs. Adopt openjdk can be recommneded in wiki.alephium.org